test/e2e/play_kube_test.go

```diff
@@ -3075,6 +3107,41 @@ o: {{ .Options.o }}`})
 		Expect(inspect.OutputToString()).To(ContainSubstring("o: " + volOpts))
 	})
 
+	It("podman play kube persistentVolumeClaim with source", func() {
```

Comment:
Shouldn't this test fail on remote?

Reply:
You're right. I changes the code to skip the test (as other tests do)

Reply:
Apologies for being vague in may last statement.

I think the test should run on remote but it _must_ fail.  So we should have something like
```go
if IsRemote() {
     // Make sure we error out with the expected error message
     return
}
// Non-remote test continues here
```

Reply:
Done